homework-002/books.rb

```diff
@@ -0,0 +1,27 @@
+# ****************************************************************************
+#
+# Book
+#
+# ****************************************************************************
+
+class Book
+  def initialize
+    @title = ''
+    @chapters = []
+  end
+
+  attr_accessor :title, :chapters
```

Comment:
because we are going to write a chapters method, we should avoid defining the  reader for it,
so just:
```ruby
attr_accessor :title
```
If you do want a chapters writer (so you could initialize a book and set its chapters later, then you can use `attr_writer` to just define a writer.